mysql-test/suite/rocksdb/include/i_s_basic.inc

```diff
@@ -0,0 +1,31 @@
+# suite/rocksdb/include/i_s_basic.inc
+#
+# SUMMARY
+#
+#    Basic MyRocks information_schema test that covers issues and sequences that
+#    are common to all of the MyRocks information_schema plugins.
+#
+# USAGE
+#
+#    let $i_s_plugin=<plugin_table_name>
+
+if (!$i_s_plugin)
+{
+  --echo i_s_plugin not specified to suite/rocksdb/include/i_s_basic.inc
+  --die i_s_plugin not specified to suite/rocksdb/include/i_s_basic.inc
```

Comment:
Just --die should be enough?

Reply:
Well, maybe it is an older/deprecated concept. I looked in some of the mysql-test/include files for some idiom to use and most did it this way, which I thought was odd, but I went with it. It is a trivial thing/change and really only there to ensure that a consumer of the .inc is using it properly.